README.md

```diff
@@ -56,6 +56,41 @@ $ run_docker_ci_job # optional (--prune)
 
 ## Langugage Spec(ish)
 
+### Types
+#### Numbets
+
+Integers are supported as numeric values. 
+
+Integers are stored as whole numbers as Rust `u64` constant for the 64-bit unsigned integers.
+
+Example:
+```
+let _a = 123;
+```
+
+#### Booleans
+
+Booleans represents a value, which could be `true` or `false`. 
+
+Example:
+```
+let _t = true;
+let _f = false;
```

Comment:
does `let _j = 1 > 2` work?

Reply:
Yes. Updated README and added a test in d9933f9e5511a77fbc85e3db373f9bbffbae616c